MoEPluginBase/MoEMessageQueueHandler.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MoEMessageQueueHandler.swift
+//  MoEPluginBase
+//
+//  Created by Rakshitha on 24/06/22.
+//
+
+import Foundation
+import MoEngageSDK
+
+@objc public protocol MoEPluginBridgeDelegate {
+    @objc func sendMessage(event: String, message: [String: Any])
+}
+
+class MoEMessageQueueHandler {
+    var identifier: String
```

Comment:
@RakshithaAcharya Add proper access modifiers for the properties

Reply:
@deepa-iOS-SDK  addressed